Documentation/guides/messages/README.md

```diff
@@ -135,6 +135,8 @@ ms.date: 01/24/2020
 + [XA5207](xa5207.md): Could not find android.jar for API level `{compileSdk}`.
 + XA5211: Embedded Wear app package name differs from handheld app package name ({wearPackageName} != {handheldPackageName}).
 + XA5213: java.lang.OutOfMemoryError. Consider increasing the value of $(JavaMaximumHeapSize). Java ran out of memory while executing '{tool} {arguments}'
++ XA5215: Duplicate Resource found for {elementName}. Duplicates are in {filenames}
++ XA5216: Resource entry {elementName} is already defined in {filename}
```

Comment:
I added these lines so I could ask if it does seem useful to add them. The idea I had was that keeping old message codes in the README could make it easier to avoid reusing them in future, and avoiding reuse would mean the web search results for the codes won't get mixed up between the old and new uses.

Another way to search for old codes is to use `git log`, but having them in the README makes it easier.

Reply:
This is a good idea, but we should probably add `[removed in Xamarin.Android 10.4]` to the end of this messages or something.